helix-core/src/test/java/org/apache/helix/integration/spectator/TestRoutingTableProviderPeriodicRefresh.java

```diff
@@ -56,7 +56,7 @@
 
   private static final String STATE_MODEL = BuiltInStateModelDefinitions.MasterSlave.name();
   private static final String TEST_DB = "TestDB";
-  private static final String CLASS_NAME = TestRoutingTableProvider.class.getSimpleName();
+  private static final String CLASS_NAME = TestRoutingTableProviderPeriodicRefresh.class.getSimpleName();
```

Comment:
getShortClassName();

Reply:
changed to `TestHelper.getTestClassName()`

getShortClassName() not working here due to this is static field and getShortClassName is not.